### PR TITLE
WIP: add astor to conda

### PR DESCRIPTION
**Pull request by drewokane**

### CODE REVIEW

**Comment by drewokane** [comment]
Would appreciate eyes on this PR. Thanks!

File: recipes/astor/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{% set name = "astor" %}
+{% set version = "0.5" %}
+{% set md5 = "0fa627a7f14e02566e941ec782c03bfc" %}
```
Comment (synapticarbors) [inline recipes/astor/meta.yaml]: I believe we prefer the sha256 (in this case `4abff251d3f03cbe99106cc16e0dc94adc4f88d580c34a717741c7090b76aab9`).

File: recipes/astor/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - astor
+
+about:
+  home: https://github.com/berkerpeksag/astor
+  license: BSD
```
Comment (synapticarbors) [inline recipes/astor/meta.yaml]: change to `BSD-3-Clause`

**Comment by drewokane** [comment]
Does anyone have insight into why only AppVeyor is failing? It keeps saying that `curl` isn't a command.

**Comment by synapticarbors** [comment]
AppVeyor changed what was on path and curl was left out. This is effecting all PRs:

https://github.com/conda-forge/conda-forge.github.io/issues/357

**Comment by drewokane** [comment]
Ready to merge.

**Pull request merged**